cardano-node/src/Cardano/Node/Configuration/Topology.hs

```diff
@@ -35,20 +36,13 @@ newtype TopologyError
   = NodeIdNotFoundInToplogyFile FilePath
   deriving Show
 
-nodeAddressToSockAddr :: NodeAddress -> SockAddr
-nodeAddressToSockAddr (NodeAddress addr port) =
-  case unNodeHostAddress addr of
-    Just (IP.IPv4 ipv4) -> SockAddrInet port $ IP.toHostAddress ipv4
-    Just (IP.IPv6 ipv6) -> SockAddrInet6 port 0 (IP.toHostAddress6 ipv6) 0
-    Nothing             -> SockAddrInet port 0 -- Could also be any IPv6 addr
-
 -- | Domain name with port number
 --
 data RemoteAddress = RemoteAddress
-  { raAddress :: !String
-  -- ^ Either a DNS address or IP address
-  , raPort :: !PortNumber
-  -- ^ Port number of the destination
+  { raAddress   :: !Text
+  -- ^ Either a dns address or ip address.
```

Comment:
Nitpick, but why change the case of DNS and IP?

Reply:
You mean why `Text` is the common denominator rather than `String`? See `remoteAddressToNodeAddress`, though using `String` is fine too.

Reply:
Oops, I meant to delete this comment because it was so insignificant.

I was referring to the change in the documentation: "DNS" was changed to "dns" and "IP"was changed to "ip".